app/views/storage_locations/show.html.erb

```diff
@@ -81,6 +81,8 @@
             <div class="tab-content" id="custom-tabs-three-tabContent">
 
               <div class="tab-pane fade show active" id="custom-tabs-inventory" role="tabpanel" aria-labelledby="custom-tabs-one-home-tab">
+                Show Inventory at Date: <%= date_field_tag 'version_date_select', params[:version_date] %>
```

Comment:
I think we've started only collecting version data recently. Could we add a message that indicates that history before that time is not populated? I think you can find that out in the commit logs and then I could tell you when it actually got deployed.

Reply:
Perfect, sure! Looks like this was merged on July 26, 2021. Thanks @edwinthinks - let me know when I can put the date as. I could even restrict the date picker so it's actually impossible to select any earlier than that date.

Reply:
The exact date this was released was Aug 2nd 2021. I think a combination of a message and restriction could be good!